x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/ml_inference_logic.ts

```diff
@@ -156,6 +160,7 @@ interface MLInferenceProcessorsActions {
   setInferencePipelineConfiguration: (configuration: InferencePipelineConfiguration) => {
     configuration: InferencePipelineConfiguration;
   };
+  startTextExpansionModelSuccess: StartTextExpansionModelApiLogicActions['apiSuccess'];
```

Comment:
@TattdCodeMonkey I believe the desired behavior is to update the list after the model download has begun (so after "create text expansion model"). A success response from that API indicates that the ELSER model placeholder has been created, so the user can start referencing it in a pipeline while the download is running.